src/main/webapp/script.js

```diff
@@ -5,15 +5,22 @@
  * @author tblanshard
  */
 
+let arePermissionsCorrect;
+
```

Comment:
Why as a global variable?

Reply:
Good point! I had originally thought it would need to be a global variable as other functions would need to check whether it was true or not, however thinking about it now, it should only call the other functions if this function returns true.